application/templates/documentation.html

```diff
@@ -16,21 +16,23 @@ <h3>GET  '/api/timers/{timer_id}'</h3>
               "work_interval": "25:00",
               "rest_interval": "5:00"
               "sound": "reverbSplash",
-              "mood": true
+              "mood": true,
+              "user_id": <user_id>
             }
           </p>
         </pre>
     </div>
-    <div class="update_timer">
-      <h3>PUT  '/api/timers/{timer_id}'</h3>
+    <div class="update_create_timer_for_user">
+      <h3>PUT  '/api/users/:user_id/timer'</h3>
+      <p>If a user already has a timer, the settings of that timer will be updated. If not, a new timer with the appropriate settings will be created.</p>
       <p class="sub_header">Request Body:</p>
         <pre>
           <p class="code_block">
             {
               "work_interval": "30:00",
               "rest_interval": "7:00",
               "sound": "gong",
-              "mood": false
+              "mood": False
```

Comment:
Ah ok, I didn't realize that when we make this PUT request, we would need to alter true/false to True/False - not sure exactly how that will work for us, since the capitalized versions would no longer be bools in JS, but I'll play around with it